### PR TITLE
New spritesheets, coins UI, and README

### PR DESCRIPTION
**Pull request by torralbaa**
Hello!

I changed the old hamster textures to the new ones, added the basic coins interface, completed the README.md file and replaced `_start_server.sh` and `_start_client.sh` with a Makefile.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make
+
+all:
+	@bundle exec jekyll serve -t --config _config.yml,_config-dev.yml &
```
Comment (tumble1999) [inline Makefile:8]: Please note that `_config-dev.yml` is only for local servers not suitable for hosted servers, I don't know how this changes this.

File: README.md [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
 
-## Getting Started
-These instructions will get you a copy of the project up and running on your local machine. See deployment for notes on how to deploy the project on a live system.
+## Getting started
 
 ### Prerequisites
-(Preperatrion to install)
+To use Private Box you need:
+* Ruby >= 2.4.0
+* Node.js >= 10.16.3
```
Comment (tumble1999) [inline README.md:11]: Also, bundle? https://rubygems.org/gems/bundler/versions/1.11.2

File: README.md [hunk 3]
```diff
@@ -1,55 +1,59 @@
...
-## Authors
-* **AuthorName** - *What they do* - [USER](https://github.com/USER)
-* **AuthorName** - *What they do* - [USER](https://github.com/USER)
-* **AuthorName** - *What they do* - [USER](https://github.com/USER)
-
-See also the list of [contributors](https://github.com/USER/REPO/contributors) who participated in this project.
+* EaselJS (CreateJS)
+* Jekyll
+* Node.js
+* Express.js
```
Comment (tumble1999) [inline README.md:40]: also socket https://www.npmjs.com/package/socket.io

**Review by tumble1999 — CHANGES_REQUESTED** [review]
I have made a review of your pull request

File: README.md (hunk 2 above)
Comment (torralbaa) [reply README.md:11]: Bundle is installed when the user runs `make deps` or `_start_client.bat`

File: README.md (hunk 3 above)
Comment (torralbaa) [reply README.md:40]: Yep. I forgot about that.

File: Makefile (hunk 1 above)
Comment (torralbaa) [reply Makefile:8]: Yes, it's true, but `_config-dev.yml` was included in `_start_client.sh`. Delete it?

Comment (tumble1999) [reply Makefile:8]: could a different param be made for each of the ways?

Comment (torralbaa) [reply Makefile:8]: Yes, something like `make` and `make dev`?

Comment (tumble1999) [reply Makefile:8]: yes, ok

Comment (torralbaa) [reply Makefile:8]: Ok, done.

**Review by tumble1999 — APPROVED** [review]
I think I approve this pull request

**Comment by torralbaa** [comment]
Thanks!

**Comment by tumble1999** [comment]
?

**Comment by tumble1999** [comment]
closed?

**Comment by torralbaa** [comment]
Sorry, I thought you had merge.

**Comment by tumble1999** [comment]
Travis was still checking that it runs ok

**Comment by tumble1999** [comment]
@Alvarito050506 i think you get buttons that you can press at the end here, but it's up to you.

**Comment by torralbaa** [comment]
@tumble1999 Yes, it's true.

I haven't added `/media/ui/coins.png` to the repo. Can you add it? Or should I create a PR?

**Pull request merged**